src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritProjectListUpdater.java

```diff
@@ -220,7 +220,7 @@ private void tryLoadProjectList() {
             }
         }
         try {
-            logger.info("Trying to load project list.");
+            logger.trace("Trying to load project list.");
```

Comment:
I like it too but if I have set project list refresh interval to 60 seconds, the log is full of this information. This information is not an error or warning. Admin can create new Log Recoreder and set Logger to com.sonyericsson.hudson.plugins.gerrit or adjust loglevel verbosity for com.sonyericsson.hudson.plugins.gerrit in global log.
But OK, I will remove it from this PR.